dependencies.mk

```diff
@@ -0,0 +1,33 @@
+SCRIPT_BUNDLER = bundle check --path=vendor/bundle || bundle install && bundle clean && rm -rf "$(BUNDLE_PATH)/$(RUBY_ENGINE_VERSION)/$(RUBY_API_VERSION)"/gems/capybara-webkit-*/src
```

Comment:
I think this is worse than it was. IMO it should be own target, properly multi line etc.

Reply:
yes, it's ugly i know. 

the thing is I am passing this as the `CMD` to the `run` target, which then: 
either runs it, in circleci case, where it's running inside container,
or passes it as run command to `docker-compose run`, in the case of jenkins. 

one minor improvement might be to inline the SCRIPT_BUNDLER in the CMD [here](https://github.com/3scale/porta/blob/24651d5a45fd5e416452d821478055310cf175d8/dependencies.mk#L6), and make that multi-line there?

Reply:
Why not have this as normal target? `make bundle`.

Reply:
yes, we do have the `bundle` target. https://github.com/3scale/porta/blob/24651d5a45fd5e416452d821478055310cf175d8/dependencies.mk#L5

maybe I'm not getting what you're suggesting?

Reply:
hmm 🤔 

in that case, how would we handle "install dependencies inside docker-compose powered env"?

...that's what this abstracts really...

Reply:
```
$ (your machine) make dev # (starts new shell)
$ (inside docker shell) make bundle
$ (inside docker shell) rake cucumber
```

Reply:
ah, yes, i think this covers local dev env... would it also work in jenkins ?

Reply:
i mean waiting for the shell and running subsequent commands in that shell... 🤔

Reply:
Why we would care about jenkins? That is going away as soon as possible.

Reply:
most probably, yes. 

before we drop support for it, I would rather wait and see how the continuous productization landscape shapes out, because that looks like it will be jenkins-based. So I would rather keep support (build passes on this PR) rather than have to implement it again from scratch.

Reply:
I don't see how is productization connected to this. Why we would be running tests in our productization process when we run them before merging to master?